Webshop/Controllers/UserController.cs

```diff
@@ -116,7 +116,7 @@ public ViewObject PostAddToList()
 
                 //User updatedUser = Auth.CurrentUser;
                 //updatedUser.MyLists.First(x => x.TitleOfList == list.TitleOfList).Games = list.Games;
-                context.Users.UpdateMyLists(Auth.CurrentUser, data.TitleOfList, game);
+                context.Users.UpdateMyLists(Auth.CurrentUser, data.TitleOfList);
```

Comment:
Moet de huidige user geupdate worden? Method is ook well erg lang door even action omdat het in deze manier lasting kan worden getest